hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+    - for free trial account on cumulocity.com you will need to request this feature by contacting
```

Comment:
Sentence above is unfinished; contacting who?

Reply:
good question, I need to confirm it